### DISCUSSION TITLE
Can't convert an animated PNG file to an animated GIF file

### DISCUSSION THREAD

**Discussion by lkintact** [ImageMagick/ImageMagick#7750] [Help] [answered]
### ImageMagick version

7.1.1-39 Q16-HDRI x64 18ae555:20241005

### Operating system, version and so on

Windows 10 Home

### Description

Hi, everyone,

I'm trying to convert a PNG animation, [animation.png](https://github.com/user-attachments/assets/bcf87e4c-0896-4df5-a78a-4d26bf0f83f8), to a GIF animation with this command:
`magick.exe animation.png animation.gif`.

The result is, however, a single-frame GIF file: [animation.gif](https://github.com/user-attachments/assets/c3204556-1875-460d-b2f2-a48d962b2c26).

Please tell me how to achieve what I want.

**Comment by snibgo** [comment, marked as answer]
Try prefixing the input with `apng:` like this, to tell IM to read all animated frames:
```
magick.exe apng:animation.png animation.gif
```

**Comment by lkintact** [comment]
Thank you, @snibgo, that worked!

Edit: for people having the same problem: notice that for the conversion to work you need the [ffmpeg tool](https://ffmpeg.org/) to be installed.
